public/v2/apps/wordpress.json

```diff
@@ -59,10 +59,17 @@
             "validRegex": "/^([^\\s^\\/])+$/"
         },
         {
-            "id": "$$cap_mysql_version",
-            "label": "MySQL Version",
+            "id": "$$cap_db_type",
+            "label": "Database Type",
+            "defaultValue": "mysql",
+            "description": "You can either choose mariadb or mysql, you need to change the version according to which DB is selected. It is case sensitive.",
+            "validRegex": "/.{1,}/"
```

Comment:
Let's change the regex to only allow mysql and mariadb

Reply:
I'm not very in regex, any clues on how to do that?

Reply:
Maybe something like this?
```
"/^(mysql|mariadb)$/"
```

Reply:
I'll try it out.

Reply:
That worked